### ISSUE TITLE
stripe.confirmPayment returns an error every time

### ISSUE THREAD

**Issue by brynn** [Fitpassu/react-native-stripe-payments#11]
I'm getting the following error whenever I try to call `stripe.confirmPayment` with my Stripe secret key and valid CC data:

```
Error: There was an unexpected error -- try again in a few seconds
    at Object.fn [as confirmPayment] (NativeModules.js:99)
    at Stripe.confirmPayment (index.js:16)
    at handleSave$ (payment-method.tsx:43)
    at tryCatch (runtime.js:45)
    at Generator.invoke [as _invoke] (runtime.js:271)
    at Generator.prototype.<computed> [as next] (runtime.js:97)
    at tryCatch (runtime.js:45)
    at invoke (runtime.js:135)
    at runtime.js:145
    at tryCallOne (core.js:37)
```

Any idea as to what could be causing this issue? Thanks!

**Comment by nlabofa** [comment]
Any luck on this ? @brynn

**Comment by brynn** [comment]
@nlabofa no - I ended up using https://tipsi.github.io/tipsi-stripe/ instead

**Comment by viktorasl** [comment]
Sorry for this late response.
I believe it was caused by wrong module name defined for Android, latest version fixes the issue 34e8918b4da17dd6faf8eb47d9d5f161e78ad4af.
